### ISSUE TITLE
About calculating the logarithm of the Gaussian mixture model

### ISSUE THREAD

**Issue by MrHhorse** [JavierAntoran/Bayesian-Neural-Networks#15]
Hi! First of all, thank you for providing your code！
I think there is an error in class "spike_slab_2GMM" in the "priors.py" file:
`normalised_like = self.pi1 + torch.exp(N1_ll - max_loglike) + self.pi2 + torch.exp(N2_ll - max_loglike)`
After calculation and verification, I think it should be changed to:
`normalised_like = self.pi1 * torch.exp(N1_ll - max_loglike) + self.pi2 * torch.exp(N2_ll - max_loglike)`
I'm sorry to disturb you in your busy schedule~

**Comment by JavierAntoran** [comment]
Hi @MrHhorse,

I agree with you. I fixed the bug. Thanks for finding that!
